great-idea/css/index.css

```diff
@@ -170,30 +172,122 @@ h5 {
 	margin: 0auto;
 }
 
-p {
-	margin: 0auto;
-}
 
 .num-email {
 	margin-top: 10px;
 	align-self: auto;
 }
 
 h6 {
-	margin-top: 50px;
+	margin-top: 100px;
```

Comment:
on this you could one line it like this 
margin: 100px 0 85px 0;
the order goes top, right, bottom, left